Rakefile

```diff
@@ -13,7 +13,7 @@ desc "Starts Sinatra and Ember servers"
 task :client_dev do
   begin
     pid = spawn("cd website && LOGSTER_ENV=development BUNDLE_GEMFILE=Gemfile bundle exec rackup --host 0.0.0.0")
-    pid2 = spawn("cd client-app && npx ember s --proxy http://localhost:9292")
+    pid2 = spawn("cd client-app && npx ember s --proxy http://127.0.0.1:9292")
```

Comment:
Might be easier to move this to a separate PR? I'm not sure if this is an individual local issue for you or a general problem...